src/main/java/seedu/staffsnap/logic/commands/ConfirmationCommand.java

```diff
@@ -0,0 +1,15 @@
+package seedu.staffsnap.logic.commands;
+import seedu.staffsnap.model.Model;
+
+/**
+ * Clears the applicant book.
+ */
+public class ConfirmationCommand extends Command {
+    public static final String COMMAND_WORD = "clear";
+    public static final String CONFIRM = "Are you sure you want to clear?";
```

Comment:
This `REQUEST_CONFIRMATION_MESSAGE` can perhaps add a visual hint for the reader to know what sort of confirmation the app is expecting. Now the user has to guess whether they should type "Y" or "y" or "YES" or "Yes" or "yes" etc.

For example, "Are you sure you want to clear all data from Staff-Snap? (Y/N)", which might be more informative.